server/.husky/pre-push

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm test
-npx lint-staged
+npm run lint:eslint
 npx pretty-quick --staged
```

Comment:
please provide command from package.json for launching pretty-quick `npm run pretty-quick`